File: frontend/app/src/main/java/com/dailymemo/domain/models/Comment.kt
package com.dailymemo.domain.models

import java.time.LocalDateTime

data class Comment(
    val id: Long,
    val memoId: Long,
    val userId: Long,
    val userName: String,
    val content: String,
    val rating: Int = 0, // 댓글 작성자의 평점 (0-5)
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime
)

File: backend/src/features/comment/model/response/comment.go
package response

import "time"

type ResComment struct {
	ID        uint      `json:"id"`
	MemoID    uint      `json:"memo_id"`
	UserID    uint      `json:"user_id"`
	UserName  string    `json:"user_name"`
	Content   string    `json:"content"`
	Rating    uint8     `json:"rating"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}

type ResCommentList struct {
	Comments []ResComment `json:"comments"`
	Total    int64        `json:"total"`
}
